### PR TITLE
version api does not report useful info

### PR DESCRIPTION
**Pull request by simong**
Calling /api/version (or /docs/rest#!/version/getVersion) produces "Could not get the branch information" in all environments I tested.

### CODE REVIEW

**Comment by simong** [comment]
I attached a commit that fixes this

File: node_modules/oae-version/lib/api.js [hunk 1]
```diff
@@ -262,19 +261,36 @@ function _getGitDescribeVersion(repo, branchName, tagsByCommitId, callback, _nrO
...
+            }, 'Unable to check if a build info file exists');
+            return callback(err);
+        } else if (exists) {
+            return callback(null, buildInfoPath);
+        } else {
+            var parent = path.dirname(directory);
+            if (parent !== directory) {
+                return _getBuildInfoFile(parent, callback);
+            } else {
+                return callback(null, null);
```
Comment (mrvisser) [inline node_modules/oae-version/lib/api.js]: Can just be `return callback()`

**Comment by mrvisser** [comment]
Looks good, just one minor style issue.

**Comment by simong** [comment]
Pushed follow-up

**Pull request merged**